playwright-tests/e2e/poms/todos.ts

```diff
@@ -54,4 +62,23 @@ export class TodoPage {
       this.page.getByTestId("tasks-pending-table").getByRole("row").nth(1)
     ).toContainText(newTodo);
   };
+
+  addCommentAndVerifyIt = async ({ comment }: AddCommentProps) => {
```

Comment:
Let's not add it at the end of variable names. This can be `addCommentAndVerify`